raiden/tests/integration/rpc/assumptions/test_rpc_gas_price_assumptions.py

```diff
@@ -202,22 +203,28 @@ def test_remote_transaction_with_zero_gasprice_is_not_mined(
     while client.block_number() < target_block_number:
         gevent.sleep(0.5)
 
-    miner_zerogas_tx = miner_client.web3.eth.getTransaction(zerogas_txhash)
-    miner_zerogas_receipt = miner_client.web3.eth.getTransactionReceipt(zerogas_txhash)
+    try:
+        miner_zerogas_tx = miner_client.web3.eth.getTransaction(zerogas_txhash)
+        miner_zerogas_receipt = miner_client.web3.eth.getTransactionReceipt(zerogas_txhash)
+    except TransactionNotFound:
+        miner_zerogas_tx = None
+        miner_zerogas_receipt = None
 
     msg = "The transaction was discarded by the miner, there is no transaction and no receipt"
     assert miner_zerogas_tx is None, msg
     assert miner_zerogas_receipt is None, msg
```

Comment:
Not new in this PR, but this message is only true if both asserts fail. Maybe we should just put a `raise` into the `except` above?

Reply:
Good point, I'll merge this and create a follow up

Reply:
Ah, we expect that both transactions aren't there. So I guess this is fine